apps/mull-ui/src/app/app.spec.tsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import App from './app';
 
+import { render } from '@testing-library/react';
 import { BrowserRouter } from 'react-router-dom';
-
-import App from './app';
+import { MockedProvider } from '@apollo/client/testing';
 
 describe('App', () => {
   it('should render successfully', () => {
     const { baseElement } = render(
-      <BrowserRouter>
-        <App />
-      </BrowserRouter>
+      <MockedProvider>
```

Comment:
What does `MockedProvider` do?

Reply:
It mocks the `ApolloProvider` used in `main.tsx`